### ISSUE TITLE
update a non-existent field in instance when updating this instance

### ISSUE THREAD

**Issue by beaulian** [marshmallow-code/marshmallow-sqlalchemy#501]
Hi,

According to the following code, when we update an instance, whatever we define other fields (not in the Model) in the Schema, these fields will be inserted to the instance, is it a bug?

https://github.com/marshmallow-code/marshmallow-sqlalchemy/blob/d4c7479b15a00abc76011b76a6b22db2386d23e1/src/marshmallow_sqlalchemy/load_instance_mixin.py#L73-L76

Thanks

**Comment by lafrech** [comment]
I would say it is a bug in client code.

Should this lib actively try to catch this?

**Comment by sloria** [comment]
i don't think we need to handle this internally. it's the user's responsibility to ensure the deserialized `data` is valid input for the model.
